### ISSUE TITLE
Resto Shaman sometimes appearing as a Resto Druid

### ISSUE THREAD

**Issue by brittyazel** [Tercioo/Details-Damage-Meter#529]
In Retail using version VWD 11011 RELEASE 10.1.0, I have hit an issue where one of my fellow healers (a Resto Shaman) is appearing sometimes as a Resto Druid. It's not always, and clearing all data will usually fix it until the point that it triggers again. I have not noticed any sort of pattern to what triggers this issue.

As you can see in the below screenshots, "Butch" is a Resto Shaman, but he is showing as a Resto Druid. (The upper screenshot is bugged, the bottom screenshot is correct.)

![Screenshot 2023-05-17 202126](https://github.com/Tercioo/Details-Damage-Meter/assets/3747347/c5738dc9-9402-4d19-a950-9b4a5681a6dc)
![Screenshot 2023-05-17 202151](https://github.com/Tercioo/Details-Damage-Meter/assets/3747347/c2fd2b38-be7e-4ee7-91f2-70febeabe61f)

**Comment by Tercioo** [comment]
Would you mind getting the latest alpha and check if the issue is gone?
"/details version" should give you "ALPHA 4"

**Comment by brittyazel** [comment]
Yeah I'll grab it, and I'll let you know if it shows up tomorrow night during raid. I don't know how to trigger it, so I'll just have to let it happen organically.

**Comment by brittyazel** [comment]
I'm on Alpha 4 now, and it's still showing the Resto Shaman as a Resto druid for a previously recorded fight. I'll report back tomorrow about fights moving forward.

**Comment by brittyazel** [comment]
I have not experienced this issue tonight. It seems like you may have fixed it

**Comment by Flamanis** [comment]
Closing this issue as it is old and could have already been fixed. If you believe this to be in error, or if the issue still persists, please create a new issue.
